plasma_framework/contracts/src/exits/utils/ExitId.sol

```diff
@@ -40,6 +41,15 @@ library ExitId {
         return _computeStandardExitId(keccak256(_txBytes), _utxoPos.outputIndex());
     }
 
+    /**
+    * @notice Given transaction bytes returns in-flight exit ID.
+    * @param _txBytes Transaction bytes.
+    * @return Unique in-flight exit id.
+    */
+    function getInFlightExitId(bytes memory _txBytes) internal pure returns (uint192) {
+        return uint192((uint256(keccak256(_txBytes)) >> 105).setBit(151));
```

Comment:
Minor: we can cast first and then setBit then we can remove one line of using Bits for uint256.
But I don't really care actually XDDD just pointing this out.
`uint192((uint256(keccak256(_txBytes)) >> 105)).setBit(151)`

Reply:
done